test/language/expressions/exponentiation/applying-the-exp-operator_A1.js

```diff
@@ -0,0 +1,26 @@
+// Copyright 2009 the Sputnik authors.  All rights reserved.
```

Comment:
these are new files for a feature previously not available.

I believe the year and author should be changed. The license file is already referenced, regarding the BSD demands.